tools/Gruntfile.js

```diff
@@ -202,11 +202,23 @@ module.exports = function(grunt) {
     gruntConfig.babel.options.inputSourceMap = grunt.file.readJSON('./temp_concat/shiny.js.map');
   });
 
+  grunt.task.registerTask(
+    "validateStringReplace",
+    "tests to make sure the version value was replaced",
+    function() {
+      var shinyContent = require('fs').readFileSync('./temp_concat/shiny.js', 'utf8');
+      if (/{{\s*VERSION\s*}}/.test(shinyContent)) {
+        grunt.fail.fatal("{{ VERSION }} was not replaced in compiled shiny.js file!")
+      }
+    }
+  );
+
   grunt.initConfig(gruntConfig);
 
   grunt.registerTask('default', [
     'newer:concat',
     'newer:string-replace',
+    'validateStringReplace',
```

Comment:
I think if you prefix this with `newer:`, it will also run when you do `grunt watch`

Reply:
watch task now executes the default task.  Before it did not run the version string replacement change.

Reply:
Ah, that makes sense. I'm glad you figured it out!